File: Iniciante/URI 2802.py
import math

resposta = None
N = None

def read_numeric():
  try:
    # read for Python 2.x
    return float(raw_input())
  except NameError:
    # read for Python 3.x
    return float(input())

N = read_numeric()
resposta = (((((math.pow(N, 4)) - 6 * (math.pow(N, 3))) + 23 * (math.pow(N, 2))) - 18 * N) + 24) / 24
print("{:0.0f}".format(resposta))

File: Iniciante/URI 2802.lua
N = tonumber(io.read())
resposta = (math.pow(N, 4.0) - (6.0 * math.pow(N, 3.0)) + (23.0 * math.pow(N, 2.0)) - (18.0 * N) + 24.0 ) / 24.0
print(string.format('%.0f', resposta))